webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala

```diff
@@ -357,54 +357,9 @@ class PrepareTemplateVariablesImpl(
     // we can't do it when we built node configuration because some (system variable at least (note: but only? If so, we could just check
     // them here, not everything).
 
-    val variables = policy.expandedVars + ((policy.trackerVariable.spec.name, policy.trackerVariable))
-
-    for {
-      stVariables <- policy.technique.getAllVariableSpecs.accumulate { variableSpec =>
-                       variableSpec match {
-                         case x: TrackerVariableSpec =>
-                           variables.get(x.name) match {
-                             case None    =>
-                               Inconsistency(
-                                 s"[${agentNodeProps.nodeId.value}:${policy.technique.id.debugString}] Misssing mandatory value for tracker variable: '${x.name}'"
-                               ).fail
-                             case Some(v) =>
-                               stVariableFromVariable(v, agentVariableHandler, agentNodeProps.nodeId, policy.technique.id)
-                                 .map(Some(_))
-                           }
-                         case x: SystemVariableSpec  =>
-                           systemVars.get(x.name) match {
```

Comment:
system variables are part of the `technique.getAllVariableSpec` method, so missing one are checked